app/models/user.rb

```diff
@@ -1,5 +1,9 @@
 class User < ApplicationRecord
```

Comment:
dont we need to add challenge and attendance associations?

Reply:
Yes, you are right. I think I lost it when my computer turned off.